Sources/BraintreePayPal/BTPayPalCheckoutRequest.m

```diff
@@ -0,0 +1,55 @@
+#import "BTPayPalCheckoutRequest_Internal.h"
+#import "BTPayPalRequest_Internal.h"
+
+@implementation BTPayPalCheckoutRequest
+
+- (instancetype)initWithAmount:(NSString *)amount {
```

Comment:
This may be outside the scope of this PR, but what do you think about removing `amount` from the constructor and setting it in the same way as the other parameters?

Reply:
Hmm.... I think the advantage of having it as part of the initializer is that you can't forget to set it. It's the only required field for this class, so it makes sense to me for the code to enforce that requirement.

That being said, it is a bit weird that this one field is different from all the rest.

Reply:
Just wanted to get thoughts on it because in Android we took it out of the constructor in favor of matching the way other fields are set.  Since it's the only required field it also makes sense to enforce setting it.

Reply:
🤔  it would be nice if iOS and Android were in parity

Reply:
Yeah we removed the amount parameter on Android because Billing Agreements don't require an amount. It's definitely required for one-time / checkout requests, but does that mean the constructor for `BTPayPalVaultRequest` would have zero parameters?

Reply:
Yep!